glossary.yml

```
@@ -6362,6 +6427,14 @@
       Una secuencia de valores, normalmente de tipo [homogéneo](#homogeneous). Los
       vectores son la estructura de datos fundamental en [R](#r_language); un [escalar](#scalar) es
       solo un vector con exactamente un elemento.
+  ja:
+    term: "ベクトル"
+    def: >
+      順番を付けられたもののリストです。
+      ベクトルの中の全てのものは[同質的な](#homogeneous)本データ型でなければいけない。
+      [R言語](#r_language)に基本的なデーター型がある。
+      ベクトルが１長さ[スカラー](#scalar)のこと。
+      ドイツ語で\「vektor\」\(ベクトル\)という。英語で\「vector\」\(ベクター\)という。
```

Was not sure the intention of this addition here, but I tried formulating it to clarify the Japanese way of pronunciation is closer to that of German rather than English.
```suggestion
      「ベクトル」はドイツ語（\「vektor\」\(ベクトル\)）発音であり、英語ではどちらかというと\「vector\」\(ベクター\)という発音に近い。
```

---

As discussed with @masamiy [for 'variable'](https://github.com/carpentries/glosario/pull/297#discussion_r609264633) we're not sure whether giving other languages is appropriate for Japanese. I think it could avoid some confusion for learners but if you disagree we can remove this.

Escaped characters removed (turns out this wasn't what caused [checks](https://github.com/carpentries/glosario/runs/2303905011) to fail).

```suggestion
      「ベクトル」はドイツ語（「vektor\」(ベクトル)）発音であり、英語ではどちらかというと「vector」(ベクター)という発音に近い。
```